etc/profile-a-l/irssi.profile

```diff
@@ -0,0 +1,58 @@
+# Firejail profile for irssi
+# Description: TUI IRC client
+# This file is overwritten after every install/update
+# Persistent local customizations
+include irssi.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.irssi
+
+# Add the next line to irssi.local if you use perl scripting.
+#include allow-perl.inc
+
+blacklist ${RUNUSER}/wayland-*
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-proc.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-X11.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/.irssi
+whitelist ${HOME}/.irssi
+include whitelist-common.inc
+include whitelist-usr-share-common.inc
+include whitelist-var-common.inc
+
+caps.drop all
+netfilter
+no3d
+nodvd
+nogroups
+noinput
+nonewprivs
+noprinters
+noroot
+nosound
+notpm
+notv
+nou2f
+novideo
+protocol unix,inet,inet6
+seccomp
+seccomp.block-secondary
+
+disable-mnt
+private-cache
+private-dev
```

Comment:
```suggestion
private-dev
private-etc @network,@tls-ca
```

Does it work with `private-etc`?

Reply:
I had installed -git to test with the `notpm` line, but now i'm back to my distro version and did not test `private-etc`. I will test with -git later on and report. But I don't see why not.

Reply:
instead I ran it now in debugger, also watching `lsof` for the pid, and nothing on etc is ever touched. I think it is OK.

Reply:
> instead I ran it now in debugger, also watching `lsof` for the pid, and
> nothing on etc is ever touched. I think it is OK.

Can you test connecting to a server in firejail-git to make sure?

It seems strange that nothing on /etc would be accessed (such as TLS-related
paths).

What is the output of trace.txt in the following?

```sh
firejail --trace=trace.txt --profile=irssi /usr/bin/irssi
```

Reply:
> My bad. My irssi profile is very uncommon. Retested this with a clean irssi
> profile, and indeed, it does break dns as expected.
> 
> with private-etc: Irssi: Unable to connect server irc.freenode.net port 6667
> [Temporary failure in name resolution]
> 
> without: resolves names fine.

Strange, what program do you use to manage dns?

On what distribution name/version?

Also, could you retest with the current irssi.profile in firejail-git (there is
one new change) and post the output of trace.txt?